lib/lightning/accounts/user_notifier.ex

```diff
@@ -155,26 +156,83 @@ defmodule Lightning.Accounts.UserNotifier do
     """)
   end
 
-  defp build_email_body(data, digest) do
-    digest_lookup = %{daily: "day", monthly: "month", weekly: "week"}
+  def build_digest_url(workflow, start_date, end_date) do
+    uri_params = %{
+      "filters" => %{
+        "body" => true,
+        "crash" => true,
+        "failure" => true,
+        "log" => true,
+        "pending" => true,
+        "search_term" => true,
+        "success" => true,
+        "timeout" => true,
+        "wo_date_after" => "",
+        "wo_date_before" => "",
+        "date_after" => start_date |> DateTime.to_string(),
+        "date_before" => end_date |> DateTime.to_string(),
+        "workflow_id" => workflow.id
+      }
+    }
```

Comment:
Why aren't we using the SearchParams struct here? 

The point of that module is to handle the defaults and add a layer of type checking to ensure that changes to the SearchParams (and/or the form) stay consistent across the app.

Reply:
Done